source/tutorials/atacama-soils.rst

```diff
@@ -93,16 +102,69 @@ You next can demultiplex the sequence reads. This requires the sample metadata f
      --p-rev-comp-mapping-barcodes \
      --i-seqs emp-paired-end-sequences.qza \
-     --o-per-sample-sequences demux.qza \
+     --o-per-sample-sequences demux-full.qza \
      --o-error-correction-details demux-details.qza
 
+Let's subsample the data. We will perform this subsampling in this tutorial
+for two reasons - one, to speed up the tutorial run time, and two, to
+demonstrate the functionality.
+
+.. warning:: The following example of subsampling reads is meant to
+   illustrate the subsampling capability of ``q2-demux``. If you are
+   considering subsampling the reads of a study, please make sure you have
+   thought it through and have reasonable justification.
+
+.. command-block::
+
+   qiime demux subsample-paired \
+     --i-sequences demux-full.qza \
+     --p-fraction 0.3 \
+     --o-subsampled-sequences demux-subsample.qza
+
    qiime demux summarize \
-     --i-data demux.qza \
-     --o-visualization demux.qzv
+     --i-data demux-subsample.qza \
+     --o-visualization demux-subsample.qzv
+
+Let's take a look at the summary in ``demux-subsample.qzv``. In the
+"Per-sample sequence counts" table on the "Overview" tab, there are 75
+samples in the data. If we look at the last 20 or so rows in the table,
+though, we will observe that many samples have fewer than 100 reads in them -
+let's filter those samples out of the data:
 
-After demultiplexing reads, we'll look at the sequence quality based on ten-thousand randomly selected reads, and then denoise the data. When you view the quality plots, note that in contrast to the corresponding plots in :doc:`the moving pictures tutorial <moving-pictures>`, there are now two interactive plots to be considered together. The plot on the left presents the quality scores for the forward reads, and the plot on the right presents the quality scores for the reverse reads. We'll use these plots to determine what trimming parameters we want to use for denoising with DADA2, and then denoise the reads using ``dada2 denoise-paired``.
+.. warning:: The following example of filtering samples is meant to
+   illustrate the filtering capability of ``q2-demux``, if you are
+   considering filtering samples out of a study, please make sure you have
+   thought it through and have reasonable justification.
+
+.. command-block::
 
-In this example we have 150-base forward and reverse reads. Since we need the reads to be long enough to overlap when joining paired ends, the first thirteen bases of the forward and reverse reads are being trimmed, but no trimming is being applied to the ends of the sequences to avoid reducing the read length by too much. In this example, the same values are being provided for ``--p-trim-left-f`` and ``--p-trim-left-r`` and for ``--p-trunc-len-f`` and ``--p-trunc-len-r``, but that is not a requirement.
+   qiime tools export \
+     --input-path demux-subsample.qzv \
+     --output-path ./demux-subsample/
+
+   qiime demux filter-samples \
```

Comment:
Sorry I missed this in the first pass, @thermokarst, but this code is failing on my machine _using 2020.2 10% data_. Doing some troubleshooting, and will edit this comment as I go.

Reply:
Confirmed: the `Sample name` column header this .tsv has appears not to be compliant for reading as metadata. Have you had different results on your machine or with your data?

Reply:
Are you running a dev env with the latest staging? The CSV->TSV change is a week or two old now.

Reply:
`make preview DEBUG=tutorials/atacama-soils` against the latest staging runs to completion.

Reply:
Mine's probably just outside two weeks. I'll bump it and see what I get. Thanks for your patience!

Reply:
All good with latest staging. Thanks for pointing me in the right direction.